proxy/coco/src/lib.rs

```diff
@@ -25,7 +25,7 @@
 
 pub use librad::{
     git::{self, identities::local::LocalIdentity, include, local::url::LocalUrl, Urn},
-    identities::{Person, Project},
+    identities::{payload as identities_payload, Person, Project},
```

Comment:
do we need this alias? I'd prefer to refer to it as `identities::payload` instead.

Reply:
Fixed